src/app/pages/datasets/components/dataset-details-panel/dataset-details-panel.component.html

```diff
@@ -25,11 +25,11 @@ <h3 class="title">
 
 <div class="cards">
   <ix-dataset-details-card [dataset]="dataset"></ix-dataset-details-card>
-  <ix-permissions-card [dataset]="dataset" *ngIf="hasPermissions"></ix-permissions-card>
+  <ix-permissions-card [dataset]="dataset" *ngIf="hasPermissions && !dataset.locked"></ix-permissions-card>
   <ix-zfs-encryption-card
-    *ngIf="dataset.encrypted"
+    *ngIf="isAllowEncryption"
     [dataset]="dataset"
     [parentDataset]="parentDataset"
   ></ix-zfs-encryption-card>
-  <ix-dataset-capacity-management-card [dataset]="dataset"></ix-dataset-capacity-management-card>
+  <ix-dataset-capacity-management-card [dataset]="dataset" *ngIf="isAllowCapacity"></ix-dataset-capacity-management-card>
```

Comment:
We will still have capacity card for zvols. If something fails in the card, for example because you can't have user/group quotas on a zvol, this needs to be fixed inside of the capacity card.